src/halo_common.f90

```diff
@@ -140,7 +140,7 @@
           !$acc end kernels
        else if (ipencil == 3) then
           jst = decomp%zst(2); jen = decomp%zen(2)
-          ist = decomp%xst(1); ien = decomp%xen(1)
+          ist = decomp%zst(1); ien = decomp%zen(1)
```

Comment:
Shall we add a CI test covering halo operations with the global flag on ?

Reply:
Ok, I will just add the description of the preprocessor variable in the md file then.